extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeGroup.php

```diff
@@ -2,14 +2,21 @@
 namespace Wikia\PortableInfobox\Parser\Nodes;
 
 class NodeGroup extends Node {
-	const DATA_LAYOUT_ATTR_NAME = 'layout';
-	const DEFAULT_TAG_NAME = 'default';
+	const LAYOUT_ATTR_NAME = 'layout';
+	const SHOW_ATTR_NAME = 'show';
+	const DEFAULT_OPTION = 'default';
+	const SHOW_INCOMPLETE_OPTION = 'incomplete';
 
 	private $supportedGroupLayouts = [
-		'default',
+		self::DEFAULT_OPTION,
 		'horizontal'
 	];
 
+	private $supportedGroupDisplays = [
+		self::DEFAULT_OPTION,
```

Comment:
I'm battling whether we should reuse DEFAULT_OPTION widely or have an explicit default for each attribute (GROUP_OPTION_DEFAULT, etc).
Thoughts?